tests/RequestParserTest.php

```diff
@@ -63,6 +63,27 @@ function testRequestViaClosure()
         $this->assertEquals('LIQRGV\QueryFilter\Mocks\MockClosureModel', $builderStruct->baseModelName);
     }
 
+    public function testModelSetter()
+    {
+        $this->markTestSkipped('Test for debug purpose only, change createModelBuilderStruct modifier to public to use');
```

Comment:
Please don't use `createModelBuilderStruct` if we're testing public method.
That one only act as verifier that our internal method work as intended.
You can see an example on `testPagination` by @rizkyarlin

Reply:
Done @LIQRGV, I have pushed a new commit to update this test 2ee4f82.